cli/src/lib/validators/validatePackageExports.js

```diff
@@ -115,8 +115,12 @@ const validateMultipleEntrypoints = (pkg, { config, paths }) => {
         exports: expectedExports,
     }
     if (expectedExports['.']) {
-        expectedPackage.module = expectedExports['.'].import
-        expectedPackage.main = expectedExports['.'].require
+        if (expectedExports['.'].import) {
+            expectedPackage.module = expectedExports['.'].import
+        }
+        if (expectedExports['.'].require) {
+            expectedPackage.main = expectedExports['.'].require
+        }
```

Comment:
This is better now that it won't erase `module` and `main` properties in `package.json`, but it also won't validate `main` and `module` unless `exports['.']` directly has `.import` and `.require` properties...

Is it important to have `main` and `module` properties? If so, you might want to follow down the `exports` properties recursively to find `import` and `require` values; if not, this is fine as it is

Reply:
Actually I may be misinterpreting this - taking another look

Reply:
> Is it important to have main and module properties?

`main` is for node versions 12 and below (which do not support `exports`), while `module` was [a proposal](https://github.com/dherman/defense-of-dot-js/blob/master/proposal.md#typical-usage) that node does not support so I think it's fine to ignore both when their 'correct' values are ambiguous.

Reply:
Okay cool :)